src/makeInfinityAdder.js

```diff
@@ -6,7 +6,27 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const add = (a) => {
```

Comment:
you should return a function that get a number. If a number exists the function updates your sum value and returns the add function. If a number doesn’t exist you reset the sum value and return the sum value before reset

for this task, you don't need a return function that returns another new function